### ISSUE TITLE
Any way to get Watchdog working with Steam Proton?

### ISSUE THREAD

**Issue by turtlshell** [Jellybaby34/Dark-Souls-3-PvP-Watchdog#23]
Just a quick question. I run Manjaro which is an Arch based distro and I play DS3 through Steam Proton.

I copied the required files over to my installation directory as normal, and it seems that the new "Intro Skip" feature actually does work which gave me some hope- I'm even able to exit out of cheater's games when I press F1 so I think everything but the top bar is fully functional! Could it be a font issue? No idea.

However there is no top bar menu to speak of, and obviously even if there was I would expect that the sound notification features, which just use default windows notification sounds, would not likely work- but that's not something I really care about.

TL;DR: Is it possible for me to get this working myself? I wouldn't want to ask you to make a separate version which works for Steam Proton, as that sounds like quite a lot of work for a niche audience.

**Comment by turtlshell** [comment]
Figured out that you need to use a d3d compiler and it works like a charm. If anyone's curious, I followed [this post](https://www.reddit.com/r/linux_gaming/comments/b2hi3g/reshade_working_in_wine_43/) on reshades which also happens to work with other dlls.

**Comment by Lignar** [comment]
@turtlshell I'm trying to use the instructions from your link but can't get this to work. Could you describe what exactly did you do? What Proton version were you using? What were your launch options?
